extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -51,29 +59,30 @@ export class BuildHelper {
 			await fs.mkdir(this.extensionBuildDir);
 		}
 
-		const buildfilesPath = await this.getBuildDirPathFromMssqlTools();
-		buildFiles.forEach(async (fileName) => {
+		// download the Microsoft.Build.Sql sdk nuget
+		const httpClient = new HttpClient();
+		const nugetPath = path.join(this.extensionBuildDir, `${fullSdkName}.nupkg`);
+		await httpClient.download(microsoftBuildSqlUrl, nugetPath, outputChannel);
+
+		// extract the files from the nuget
+		await extractZip(nugetPath, { dir: path.join(this.extensionDir, buildDirectory, sdkName) });
```

Comment:
if the download or extraction fails should there be some error handling?  I don't see return value checks so I guess there is a try/catch that will report the error and recover?

Reply:
yup the errors were previously getting surfaced to the user in an error notification, but I added try catches around the download and extract steps with more clear error messages.